.github/workflows/migration-check.yaml

```diff
@@ -53,14 +53,31 @@ jobs:
           features: postgres
           args: "--no-default-features"
 
-      - name: Verify `diesel migration run`
+      - uses: baptiste0928/cargo-install@v2.2.0
+        with:
+          crate: just
+          args: "--no-default-features"
+
+      - name: Verify `diesel migration run for v1`
+        shell: bash
+        env:
+          DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres
+        run: just migrate run --locked-schema --all
+
+      - name: Verify `diesel migration redo for v1`
+        shell: bash
+        env:
+          DATABASE_URL: postgres://postgres:postgres@localhost:5432/postgres
+        run: just migrate redo --locked-schema --all
+
+      - name: Verify `diesel migration run for v2`
```

Comment:
Are we expected to have a fresh database before running v2 migrations? If yes, shouldn't we be re-creating the database before this?